package.json

```diff
@@ -21,5 +21,9 @@
   },
   "dependencies": {
     "musicmetadata": "^2.0.5"
+  },
+  "contributors": {
+    "name": ["Adam Palazzo", "Juliana Dixon"],
```

Comment:
This isn't the format for multiple contributors. Can you look at the spec again?